internal/wasm/jit/compiler.go

```diff
@@ -357,4 +357,19 @@ type compiler interface {
 	//
 	// https://www.w3.org/TR/2022/WD-wasm-core-2-20220419/appendix/changes.html#bulk-memory-and-table-instructions
 	compileMemoryFill() error
+	// compileTableInit adds instructions to perform operations corresponding to the wasm.OpcodeTableInit instruction in
+	// the bulk-memory-operations proposal.
```

Comment:
I know I started this, but we have constants so we can probably use them to help centralize docs. Ex. 
```suggestion
	// wasm.FeatureBulkMemoryOperations.
```

then if that symbol already has the link you don't need to repeat it under!